__tests__/api.test.js

```diff
@@ -155,4 +155,33 @@ describe(' 6. GET /api/reviews/:review_id/comments', () => {
                 expect(res.body.msg).toBe('Ooops, nothing to see here!')
             })
     });
+});
+
+describe('7. POST /api/reviews/:review_id/comments ', () => {
+    test('successfully posts comment', () => {
+        return request(app)
+            .post('/api/reviews/1/comments')
+            .send({
+                username: 'dav3rid',
+                body: 'it was nice'
+            })
+            .expect(201)
+            .then((res) => {
+                expect(Object.keys(res.body)).toEqual(['comment_id', 'body', 'review_id', 'author', 'votes', 'created_at'])
+            })
+        
+    });
+
+    test('errors with non existing user', () => {
+        return request(app)
+            .post('/api/reviews/1/comments')
+            .send({
+                username: 'cool_username',
+                body: 'this is a review'
+            })
+            .expect(400)
+            .then((res) => {
+                expect(res.body.msg).toBe('Bad request')
+            })
```

Comment:
Solid error test. You should add a few more
- missing key on request body
- invalid review id
- non existent review id